_code-samples/get-started/go/base/ws/main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+    "fmt"
+
+    "github.com/Peersyst/xrpl-go/xrpl/websocket"
+)
+
+func main() {
+
+    // Define the network client
+    client := websocket.NewClient(websocket.NewClientConfig().
+        WithHost("wss://s.altnet.rippletest.net:51233"))
```

Comment:
Shouldn't we add the `WithFaucetProvider()` function here since we are connecting to Testnet?

```suggestion
	client := websocket.NewClient(websocket.NewClientConfig().
		WithHost("wss://s.altnet.rippletest.net:51233").
		WithFaucetProvider(faucet.NewTestnetFaucetProvider()),
	)
```